README.md

```diff
@@ -577,20 +443,234 @@ anything. If a company exits at a very high valuation relative to what was expec
+
+The purpose of equity compensation is threefold:
+
+- To attract the best talent:
+  When a company already has or can be predicted to have significant financial success, the
+  most talented people are incentivized to work for the company by the prospect of their
+  equity turning into large amounts of cash in the future.
+  The actual probability of life-changing lucre may be low (or at least, lower than you may
+  think if your entire knowledge of startups is watching
+  [*The Social Network*](https://en.wikipedia.org/wiki/The_Social_Network)). But even a small
+  chance at “winning big” can itself be exciting or worth a lot to some people.
+- To align individuals’ incentives with the interests of the company:
+  Even companies that can afford to pay lots of cash may prefer to give employees equity, so
+  that employees work to increase the *future* value of the company.
+  At its best, this philosophy fosters an environment of teamwork and a “rising tides lift
+  all boats” mentality.
+  It also encourages longer-term thinking, which is key for company success.
+  As we’ll [discuss later](#offers-and-negotiations), the amount of equity you’re offered
+  usually reflects both your past or current contribution to the company and your commitment
+  to the company in the future.
+- To reduce the company’s spending on cash compensation:
+  By giving equity, a company can often pay less cash to employees now, with the hope of
+  rewarding them later, and put that money toward other investments or operating expenses.
+  This can be essential in the early stages of a company or at other times where there may
+  not be enough revenue to pay large salaries.
+  It can also help recruit senior employees or executives who would otherwise command
+  extraordinarily high salaries.
+
+### What is equity worth?
+
+The key question to anyone who owns equity is, what is it worth, now or later?
+
+- With public companies, you can usually sell any stock you won outright, so the answer is
+  relatively easy:
+  You look up the current market value of the stock.
+  - 🚧 Mention lockup periods etc.
+- ❗️ With private companies, it’s hard to know the “real value” of stock and stock options.
+  Because the value of private company stock is not determined by public trades,
+  shareholders can only make educated guesses about the likely future value, at a time when
+  they will be able to buy or sell the stock.
+  It’s always worth remembering private company stock is simply a legal agreement that
+  entitles you to something of highly uncertain value, and could well be worthless in the
+  future, or highly valuable, depending on the fate of the company.
+- ☝️ We’ll discuss the notion of [fair market value](#409a-valuations) later, but even if a
+  company assigns a value to the stock for tax and accounting purposes, it doesn’t mean you
+  can expect to sell it for that value!
+- 🄳 The ability to buy and sell stock is called **liquidity**. In startups and many private
```

Comment:
Is there a good link for this? I'm having trouble finding this definition of liquidity. I see it referenced WTR stocks when the fast sale of stocks does not affect their price.

Reply:
This is a larger article but the leading definition is the same. https://trendshare.org/how-to-invest/what-is-liquidity